sdk/nodejs/src/client.ts

```diff
@@ -296,6 +305,35 @@ export class DevCycleClient {
         this.eventQueue.queueEvent(populatedUser, event)
     }
 
+    private trackSDKConfigEvent(
+        url: string,
+        responseTimeMS: number,
+        res?: Response,
+        err?: ResponseError,
+        reqEtag?: string,
+        reqLastModified?: string,
+    ): void {
+        const populatedUser = DVCPopulatedUserFromDevCycleUser({
+            user_id: `${this.clientUUID}@${this.hostname}`,
+        })
+
+        this.eventQueue.queueEvent(populatedUser, {
+            type: 'sdkConfig',
+            target: url,
+            value: responseTimeMS,
```

Comment:
given this may not be available in other sdks - do we want it as optional, or required here?

Reply:
this is in the NodeJS SDK, it would always be available here.